misk-tailwinds/build.gradle.kts

```diff
@@ -0,0 +1,29 @@
+plugins {
+  kotlin("jvm")
+  `java-library`
+}
+
+dependencies {
+  api(Dependencies.kotlinxHtml)
+  api(Dependencies.wispDeployment)
+  api(project(":misk-hotwire"))
+
+
+//  implementation(Dependencies.guice)
```

Comment:
remove all of these commented out deps. you also need to add a new section to each `build.gradle.kts` with
```
configure<MavenPublishBaseExtension> {
  configure(
    KotlinJvm(javadocJar = Dokka("dokkaGfm"))
  )
}
```

and `id("com.vanniktech.maven.publish.base")` in the plugins list (look at the other build.gradle files for examples)